nncf/experimental/torch/fx/constant_folding.py

```diff
@@ -0,0 +1,266 @@
+# Copyright (c) 2024 Intel Corporation
```

Comment:
I don't find where you use constant folding transformation.

Reply:
https://github.com/openvinotoolkit/nncf/pull/2996/files#diff-76750172cf1be3020f4d3a5a71609a952f1c009e24960424d4d02175613317a5R82-R83

Reply:
Could you explain why you use the constant folding transformation only in tests. How will user apply the constant folding transformation to the model?

Reply:
Fixed, please check